src/kw_config_loader.sh

```diff
@@ -27,14 +28,41 @@ function show_variables()
     has_local_config_path="No"
   fi
 
+  if [[ "$1" == 'TEST_MODE' ]]; then
+    test_mode='1'
+  fi
+
+  local -A descriptions=(
+    [arch]='Target arch'
+    [mount_point]='Mount point'
+    [virtualizer]='Virtualization tool'
+    [qemu_hw_options]='QEMU options'
+    [qemu_net_options]='QEMU Net options'
+    [ssh_ip]='Default ssh ip'
+    [ssh_port]='Default ssh port'
+    [kernel_img_name]='Kernel image name'
+    [cross_compile]='Cross-compile name'
+    [menu_config]='Default kernel menu config'
+    [qemu_path_image]='Default virtualization tool'
+    [alert]='Default alert options'
+    [sound_alert_command]='Command for sound completion alert'
+    [visual_alert_command]='Command for visual notification of completion'
+    [doc_type]='Command to generate kernel-doc'
+    [default_deploy_target]='Deploy default target'
+    [reboot_after_deploy]='Reboot after deploy'
+    [disable_statistics_data_track]='Disable tracking of statistical data'
+    [gui_on]='Command to activate GUI'
+    [gui_off]='Command to deactivate GUI'
+  )
+
   say "Variables:"
   echo -e "\tLocal config file: $has_local_config_path"
-  echo -e "\tTarget arch: ${configurations[arch]}"
-  echo -e "\tMount point: ${configurations[mount_point]}"
-  echo -e "\tVirtualization tool: ${configurations[virtualizer]}"
-  echo -e "\tQEMU options: ${configurations[qemu_hw_options]}"
-  echo -e "\tQEMU Net options: ${configurations[qemu_net_options]}"
-  echo -e "\tVdisk: ${configurations[qemu_path_image]}"
+
+  for option in "${!descriptions[@]}"; do
+    if [[ -v configurations[$option] || "$test_mode" == '1' ]]; then
+      echo -e "\t${descriptions[$option]} ("$option"): ${configurations[$option]}"
```

Comment:
IMO, replace `\t` for two spaces will look better.

Reply:
agree!